draft/mailing_list.rst

```diff
@@ -0,0 +1,19 @@
+2. Mailing lists are now only detected by the ListId header, because how should DC sort mailing lists if the ListId header is unset and only the precedence header says it's a mailing list (?). We could just hide these emails (with "junk" or "list" precedence), as we did before, if there is a reason to do so. 
+
+   I do not actually understand why such emails were hidden in the first place, though; if there is an automatic answer stating that someone is out of office or if an email could not be delivered (these are the usual reasons why such emails are sent), I would want to know about this as a user.
```

Comment:
@r10s wrote:

> [...] keep ignoring mails with Precedence-header

Why? I do not get many emails with Precedence header that are not mailing list and if I do get them I want to know about it (out-of-office-notifications etc.).